src/Sylius/Bundle/ApiBundle/StateProcessor/Admin/AdminUser/PersistProcessor.php

```diff
@@ -22,7 +21,7 @@
 use Webmozart\Assert\Assert;
 
 /** @implements ProcessorInterface<AdminUserInterface> */
-final readonly class AdminUserProcessor implements ProcessorInterface
+final readonly class PersistProcessor implements ProcessorInterface
```

Comment:
This one is only for `PUT` operations, so I wonder if it shouldn't be called
```suggestion
final readonly class PutProcessor implements ProcessorInterface
```
or
```suggestion
final readonly class PutPersistProcessor implements ProcessorInterface
```
🤔

Reply:
I like this concept. It would add more meaning to the services:
1. Persist - for both PUT and POST operations
2. PutPersist - solely for PUT operation
3. PostPersist - solely for POST operation

Reply:
👎 Either this would be a break from convention or we'd need to apply this to every single other processor we have and we'd be back to (put/patch/post/delete) processors.
Furthermore, when it's used is ultimately up to the person implementing it (in the userland as well), I don't see any major problems in someone using this processor for patch or post.